src/acc/dbcsr_acc_timings.F

```diff
@@ -47,10 +47,10 @@ END FUNCTION dbcsr_timestop
    SUBROUTINE dbcsr_timeset_F(routineN, routineN_len, handle) BIND(C, name="dbcsr_timeset")
 
       TYPE(C_PTR), INTENT(IN)                :: routineN
-      INTEGER(KIND=C_INT), INTENT(IN)        :: routineN_len
-      INTEGER(KIND=C_INT), INTENT(OUT)       :: handle
+      INTEGER, INTENT(IN)        :: routineN_len
+      INTEGER, INTENT(OUT)       :: handle
```

Comment:
I'd say that declaring it as a C_INT would have been correct, but why not taking it as `VALUE` instead of a pointer?

Reply:
Hmmm ... thank you for that comment. You're right, I'll take them as a value.

PS: my Fortran skills are very subpar and I'm just trying my best 😉